Invocation_OnBackupLeftTest.whenPrimaryResponseAlreadyReceived

```
java.lang.AssertionError: null
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertNotNull(Assert.java:712)
	at org.junit.Assert.assertNotNull(Assert.java:722)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_OnBackupLeftTest$2.run(Invocation_OnBackupLeftTest.java:120)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:1158)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:1175)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_OnBackupLeftTest.waitForPrimaryResponse(Invocation_OnBackupLeftTest.java:117)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_OnBackupLeftTest.whenPrimaryResponseAlreadyReceived(Invocation_OnBackupLeftTest.java:98)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-IbmJDK1.8/com.hazelcast$hazelcast/862/testReport/junit/com.hazelcast.spi.impl.operationservice.impl/Invocation_OnBackupLeftTest/whenPrimaryResponseAlreadyReceived/

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-freestyle/1610/testReport/junit/com.hazelcast.spi.impl.operationservice.impl/Invocation_OnBackupLeftTest/whenPrimaryResponseAlreadyReceived/

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-OracleJDK1.7-freestyle/123/testReport/junit/com.hazelcast.spi.impl.operationservice.impl/Invocation_OnBackupLeftTest/whenPrimaryResponseAlreadyReceived/

```
java.lang.AssertionError: null
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertNotNull(Assert.java:712)
	at org.junit.Assert.assertNotNull(Assert.java:722)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_OnBackupLeftTest$2.run(Invocation_OnBackupLeftTest.java:120)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:1158)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:1175)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_OnBackupLeftTest.waitForPrimaryResponse(Invocation_OnBackupLeftTest.java:117)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_OnBackupLeftTest.whenPrimaryResponseAlreadyReceived(Invocation_OnBackupLeftTest.java:98)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-OracleJDK1.6-freestyle/381/testReport/junit/com.hazelcast.spi.impl.operationservice.impl/Invocation_OnBackupLeftTest/whenPrimaryResponseAlreadyReceived/

--- a/hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/Invocation_OnBackupLeftTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/Invocation_OnBackupLeftTest.java
@@ -38,9 +38,10 @@ import java.io.IOException;
 import java.util.Set;
 import java.util.concurrent.ConcurrentHashMap;
 
+import static com.hazelcast.spi.impl.operationservice.impl.InvocationConstant.VOID;
 import static com.hazelcast.util.UuidUtil.newUnsecureUuidString;
 import static java.util.Collections.newSetFromMap;
-import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNotEquals;
 import static org.junit.Assert.assertTrue;
 
 @RunWith(HazelcastParallelClassRunner.class)
@@ -117,7 +118,7 @@ public class Invocation_OnBackupLeftTest extends HazelcastTestSupport {
         assertTrueEventually(new AssertTask() {
             @Override
             public void run() {
-                assertNotNull(f.invocation.pendingResponse);
+                assertNotEquals(VOID, f.invocation.pendingResponse);
             }
         });
     }

